XamarinCommunityToolkit/Converters/InvertedBoolConverter.shared.cs

```diff
@@ -17,7 +17,8 @@ public class InvertedBooleanConverter : IValueConverter
         /// <param name="parameter">Additional parameter for the converter to handle. Not implemented.</param>
         /// <param name="culture">The culture to use in the converter.</param>
         /// <returns>An inverted boolean from the one coming in.</returns>
-        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => !(bool)value;
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+            => !(value as bool?);
```

Comment:
if the value is **null* than "null" will be returned as the result from the method.
Do we actually want to have this behavior?

Reply:
If a user provides something that's not a boolean we cannot invert it. In my opinion, we shouldn't return `false` as that suggests some inversion actually took place but it hasn't. Returning null at least indicates that it couldn't be inverted and people would be triggered earlier that something is going wrong without immediately crashing their app.

Reply:
Got your point.
So we decided to go ahead with "no crash" approach.
I don't want to stop the works and if everyone is fine with this approach, then I am fine too.

@jfversluis @roubachof @Depechie

Reply:
I'm still pretty convinced that we should log any error to output windows. It could lead to very painful debugging when some code silently fails.

Reply:
Yeah that would certainly be nice. There was something in place for that, but not sure if that still/has ever worked perfectly.

This was added, but I think I also saw an issue of it not working: https://blog.verslu.is/xamarin/xamarin-forms-xamarin/logwarningstoapplicationoutput-xamarin-forms/

I think this should still work: https://blog.verslu.is/xamarin/xamarin-forms-xamarin/binding-errors-output-window/

Reply:
My point mainly is that we need to make a choice together _or_ we pick something and let the community decide later if that has their majority vote. Else we'll be sitting here with open PRs until eternity 😂

Reply:
Then I would either say raise exception or figure out how to do the logging to the output window. The first option is probably fastest to implement at this point.

Users will be aware that something is up with their code if it fails

Reply:
@sthewissen :) agreed.

My main concern about this PR is that **null** equals **null** :) That's only one thing that stops me to approve this PR